addons/account/views/account_move_views.xml

```diff
@@ -464,10 +464,10 @@
             <field name="arch" type="xml">
                 <tree string="Journal Entries" sample="1" decoration-info="state == 'draft'" expand="context.get('expand', False)">
                     <field name="company_currency_id" column_invisible="True"/>
-                    <field name="made_sequence_hole" column_invisible="True"/>
+                    <field name="made_sequence_gap" column_invisible="True"/>
                     <field name="invoice_date" string="Invoice Date" optional="hide" readonly="state != 'draft'"/>
                     <field name="date" readonly="state in ['cancel', 'posted']"/>
-                    <field name="name" decoration-danger="made_sequence_hole"/>
+                    <field name="name" decoration-danger="made_sequence_gap and state == 'posted'"/>
```

Comment:
I'm making a difference between gap because draft or gap because worse issue.
The entire line will be blue if we rest to draft, I think that having the sequence in red is a bit too much.
Note that you can't lock your period with draft entries anyway.